app/src/renderer/windows/account-manager/style.css

```
@@ -390,13 +504,21 @@
 }
 
 .account-list {
+  --account-list-scrollbar-gap: 0.75rem;
+  --account-list-scrollbar-clearance: 0.875rem;
+
   align-content: start;
   display: grid;
   gap: 0.875rem;
   grid-template-columns: repeat(2, minmax(18rem, 1fr));
+  margin-right: calc(
+    (var(--account-list-scrollbar-clearance) + var(--account-list-scrollbar-gap)) *
```

Stylelint (17.12.0)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)